test/runnable/iasm.d

```diff
@@ -2775,7 +2775,7 @@ L1:
     /*
     for (;;)
     {
-        writef("");
```

Comment:
Are those `printf`'s required to avoid optimizations?

Reply:
DMD doesn't optimize `asm` blocks.

Reply:
Right. Is there a specific reason to have those elaborate NOOPs?

Reply:
Ask the author @WalterBright, the entire file was added back in 2011.

Reply:
I don't see any NOOPs in the file. As for this function, it's the example from a bug report long ago.